pkgs/applications/misc/authenticator/default.nix

```diff
@@ -80,5 +78,6 @@ stdenv.mkDerivation rec {
     license = lib.licenses.gpl3Plus;
     maintainers = with lib.maintainers; [ austinbutler ];
     platforms = lib.platforms.linux;
+    broken = stdenv.isLinux && stdenv.isAarch64;
```

Comment:
Please add a very short comment why it was marked broken,

Reply:
Added.